app/views/data_tables/plant_lines_table.html.haml

```diff
@@ -0,0 +1,10 @@
+= render 'data_tables/tabs'
+
+.lead Plant Lines table
+
+= link_to 'TEMP LINK TO Plant populations', data_tables_path(model: :plant_populations)
+
+= datatable_tag do
+  - PlantLine.table_columns.each do |table_column|
```

Comment:
Right, it was such a long read that I forgot you mentioned it ;)

2015-04-02 21:02 GMT+02:00 Tomasz Gubała <notifications@github.com>:

> In app/views/data_tables/plant_lines_table.html.haml
> <https://github.com/eSpectrum-IT/brassica/pull/108#discussion_r27686944>:
>
> > @@ -0,0 +1,10 @@
> > += render 'data_tables/tabs'
> > +
> > +.lead Plant Lines table
> > +
> > += link_to 'TEMP LINK TO Plant populations', data_tables_path(model: :plant_populations)
> > +
> > += datatable_tag do
> > +  - PlantLine.table_columns.each do |table_column|
>
> I know, but as I mentioned in the PR description, I deliberately don't do
> it (yet).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/eSpectrum-IT/brassica/pull/108/files#r27686944>.
>



-- 
Tomasz Szymczyszyn